mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/print/config.yaml

```diff
@@ -37,7 +37,7 @@ scales:
   - 559082264
 
 # see https://github.com/georchestra/georchestra/issues/362
-brokenUrlPlaceholder: http://sdi.georchestra.org/public/transparent.gif
+brokenUrlPlaceholder: https://sdi.georchestra.org/public/transparent.gif

```

Comment:
> More relevant point (since the doc mentionned letsencrypt above): The letsencrypt certificates might not be trusted by the JVM: http://stackoverflow.com/questions/34110426/does-java-support-lets-encrypt-certificates

Argh. That means if we want everyone to be able to communicate (ie harvest, use wxs services..) with https-only-with-letsencrypt-cert instances, we have to document/explain how to import LE's root cert in their tomcat's keystore, since we cant expect/rely on having java8 ?